sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/UpdateCenterDeserializer.java

```diff
@@ -103,23 +106,27 @@ public UpdateCenter fromManyFiles(File mainFile) throws IOException {
     try (InputStream in = Files.newInputStream(mainFile.toPath())) {
       Properties props = new Properties();
       props.load(in);
-      loadPluginProperties(mainFile, props);
+      loadProperties(mainFile, props, PLUGINS);
+      loadProperties(mainFile, props, SCANNERS);
       UpdateCenter pluginReferential = fromProperties(props);
       pluginReferential.setDate(new Date(mainFile.lastModified()));
       return pluginReferential;
     }
   }
 
-  private static void loadPluginProperties(File file, Properties props) throws IOException {
-    String[] pluginKeys = getArray(props, PLUGINS);
-    for (String pluginKey : pluginKeys) {
-      File pluginFile = new File(file.getParent(), pluginKey + ".properties");
-      try (InputStream pluginFis = Files.newInputStream(pluginFile.toPath())) {
-        Properties pluginProps = new Properties();
-        pluginProps.load(pluginFis);
-        for (Map.Entry<Object, Object> prop : pluginProps.entrySet()) {
-          props.put(pluginKey + "." + prop.getKey(), prop.getValue());
+
+
+  private static void loadProperties(File file, Properties props, String listKey) throws IOException {
+    String[] keys = getArray(props, listKey);
+    for (String key : keys) {
+      File scannerFile = new File(file.getParent(), key + ".properties");
+      try (InputStream pluginFis = Files.newInputStream(scannerFile.toPath())) {
+        Properties p = new Properties();
+        p.load(pluginFis);
+        for (Map.Entry<Object, Object> prop : p.entrySet()) {
+          props.put(key + "." + prop.getKey(), prop.getValue());
```

Comment:
You're mixing "scanner" and "plugin" wording in this method. As it's an abstract method, I would remain consistent, and use something else (component?)